### PR TITLE
fixed location of overrides

### PR DESCRIPTION
**Pull request by krizic**
#### Description of what you did:
fixed https://github.com/strapi/strapi/issues/4499

#### My PR is a:

- [ ] 💥 Breaking change
- [x] 🐛 Bug fix
- [ ] 💅 Enhancement
- [ ] 🚀 New feature

#### Main update on the:

- [ ] Admin
- [ ] Documentation
- [ ] Framework
- [x] Plugin

#### Manual testing done on the following databases:

- [x] Not applicable
- [ ] MongoDB
- [ ] MySQL
- [ ] Postgres
- [ ] SQLite

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,6 +25,7 @@
     "npm-run-all": "^4.1.5",
     "prettier": "^1.18.2",
     "react-test-renderer": "^16.9.0",
+    "redux-saga": "~1.1.3",
```
Comment (soupette) [inline package.json]: @krizic why did you add `redux-saga` as a dependency to the monorepo?

Comment (krizic) [reply package.json]: was failing on commit hook - (using node v10.17.0 (npm v6.11.3))

Comment (krizic) [reply package.json]: removed, since it didn't belong to this issue
